tests/basic/vcluster_test.go

```diff
@@ -0,0 +1,149 @@
+package tests
+
+import (
+	"fmt"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/portworx/torpedo/drivers/vcluster"
+	"github.com/portworx/torpedo/pkg/log"
+	"github.com/portworx/torpedo/tests"
+	. "github.com/portworx/torpedo/tests"
+	"os"
+	"os/exec"
+	"strings"
+	"time"
+)
+
+var _ = Describe("VCluster", func() {
+	JustBeforeEach(func() {
+		StartTorpedoTest("VclusterOperations", "Create, Connect and execute a method on Vcluster", nil, 0)
+	})
+	//var vclusterNames = []string{"my-vcluster1", "my-vcluster2", "my-vcluster3"}
+	var vclusterNames = []string{"my-vcluster1"}
+	It("Create and connect to vclusters and run a sample method", func() {
+		Step("Create vClusters", func() {
+			for _, name := range vclusterNames {
+				err := vcluster.CreateVCluster(name)
+				Expect(err).NotTo(HaveOccurred())
+			}
+		})
+		Step("Wait for all vClusters to come up in Running State", func() {
+			for _, name := range vclusterNames {
+				err := vcluster.WaitForVClusterRunning(name, 10*time.Minute)
+				Expect(err).NotTo(HaveOccurred())
+			}
+		})
+		Step("Connect to each vCluster and execute a method", func() {
+			for _, name := range vclusterNames {
+				log.Infof("Trying to connect to %v", name)
+				err := ConnectVClusterAndExecute(name, SampleMethod, name)
+				if err != nil {
+					log.FailOnError(err, fmt.Sprintf("Failed to connect and execute method on cluster %v", name))
+				}
+			}
+		})
+	})
+	JustAfterEach(func() {
+		defer EndTorpedoTest()
+		for _, name := range vclusterNames {
+			err := vcluster.DeleteVCluster(name)
+			if err != nil {
+				log.Errorf("Error deleting Vcluster with name %v", name)
```

Comment:
Return error directly. In this case your error would be: 
Error deleting Vcluster with name name <name> 
But your DeleteVCluster is already returning: Failed to delete vcluster <name>